### PR TITLE
Changes for level UX game screen controls

### PR DESCRIPTION
**Pull request by shubhi1092**
Involves changes in the game screen's controls (Level playback view), as well as some minor styling changes for other things in the play level view.

To test manually, you can go to any level via URL `/ozaria/play/level/:slug` and notice the change in the control bar below the game.

### CODE REVIEW

File: ozaria/site/views/play/level/LevelPlaybackView.coffee [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require('ozaria/site/styles/play/level/level-playback-view.sass')
```
Comment (shubhi1092) [inline ozaria/site/views/play/level/LevelPlaybackView.coffee]: Some code has been deleted from this file to get rid of the popup that used to appear on hover of the scrubber in the legacy play level view.

File: ozaria/site/styles/play/level/level-playback-view.sass [hunk 2]
```diff
@@ -3,18 +3,16 @@
 @import "ozaria/site/styles/play/variables"
 
 #playback-view
-  $playback-button-color: rgb(248, 197, 146)
+  $playback-button-color: #ffffff
```
Comment (AndrewJakubowicz) [inline ozaria/site/styles/play/level/level-playback-view.sass]: This is a variable declaration in sass. Thanks @shubhi1092 for telling me this offline.

File: ozaria/site/styles/play/level/level-playback-view.sass [hunk 3]
```diff
@@ -129,16 +122,15 @@
...
+        background: #9b9b9b
+        border-radius: 7px
 
         .scrubber-handle
           cursor: pointer
           position: absolute
           right: -18px
           top: -11px
-          background: transparent url(/images/level/scrubber_knob.png)
+          background: transparent url(/images/ozaria/level/scrubber_knob.png)
```
Comment (AndrewJakubowicz) [inline ozaria/site/styles/play/level/level-playback-view.sass]: This is fantastic!

File: ozaria/site/templates/play/play-level-view.pug [hunk 4]
```diff
@@ -42,22 +41,10 @@ include game-dev-goals
...
-
       #playback-view
 
-      #thang-hud
-
-      #level-dialogue-view
-
       button.btn.btn-lg.btn-primary.banner.header-font#stop-cinematic-playback-button(data-i18n="[title]common.continue;common.continue")
 
-    #surface-context-menu-view
```
Comment (AndrewJakubowicz) [inline ozaria/site/templates/play/play-level-view.pug]: These removals are very good and very interesting. Should we remove all of the associated component?

For example. In this instance should [this code path](https://github.com/codecombat/codecombat/blob/b164782606b40a5c116e452ea6a21e811281c394/ozaria/site/views/play/level/PlayLevelView.js#L142) be cleaned up?

Non blocking. I think this is an excellent refactoring and wondering if it's in scope to delete more. Or if that's something we should do incrementally.

Comment (shubhi1092) [reply ozaria/site/templates/play/play-level-view.pug]: Yeah, I agree that there is a lot of associated code for these removed elements, and I have just removed them from the jade file for now. I think we can delete the related code incrementally as and when appropriate. A lot of legacy clean up will follow the level UX re-design (will note this down for later). Thanks for pointing this out! :)

**Pull request merged**